FrontEnd/scripts/works.js

```diff
@@ -50,31 +56,73 @@ async function displayWork(worksCategory) {
     workImage.setAttribute("alt", work.title);
     workFigCaption.textContent = work.title;
 
-    // NOTE: adds to elements to the parent html container "gallery" cold before
+    // NOTE: adds to elements to the parent html container called before
     workFigure.appendChild(workImage);
     workFigure.appendChild(workFigCaption);
-    worksContainer.appendChild(workFigure);
+    target.appendChild(workFigure);
   });
 }
 
+// NOTE: Ici commence l'execution du code
+//
+//
+if (sessionStorage.getItem("token") != null) {
```

Comment:
The idea is to check:
- token is not null (done)
- token is not an empty string (todo)